pkg/sql/schemachanger/scbuild/internal/scbuildstmt/alter_table_add_column.go

```diff
@@ -232,8 +232,43 @@ func alterTableAddColumn(
 				idx.KeyColumnIDs = append(idx.KeyColumnIDs, namesToIDs[colName])
 			}
 		}
-		addSecondaryIndexTargetsForAddColumn(b, tbl, idx, primaryIdx)
+		secondaryIndexIDs := addSecondaryIndexTargetsForAddColumn(b, tbl, idx, primaryIdx)
+		indexIDsForZoneConfig = append(indexIDsForZoneConfig, secondaryIndexIDs...)
+	}
+	{
+		tableElts := b.QueryByID(tbl.TableID)
+		_, _, tblNS := scpb.FindNamespace(tableElts)
+		if _, _, elem := scpb.FindTableLocalityRegionalByRow(tableElts); elem != nil {
+			_, _, currentZoneConfigElem := scpb.FindTableZoneConfig(tableElts)
```

Comment:
I wasn't able to find where `scpb.FindTableZoneConfig` is defined either in master or in this pr :(
But...can a multiregion table have zero or more than one zone config?

Reply:
nvm...my bad..github collapses the file heh..